ebb_lint/test/test_flake8.py

```diff
@@ -400,6 +400,30 @@ class C(object):
 $L204$'a' 'b'  $L303$# noqa
 
     ''',
+
+    # Oh boy these following two cases need explanation. The prefix of the
+    # 'spam' node in both of these cases ends with a literal \ followed by a
+    # newline, which cannot be tokenized by generate_tokens because it expects
+    # something to follow that sequence. This is one hell of a special case,
+    # but it made flake8 fail hard on valid syntax. The first case is a prefix
+    # containing no comment, and the second case also includes a comment which
+    # requires scanning.
```

Comment:
@habnabit , does this occur on all versions of Python?  If it is specific versions, this blurb should also note that.  If it is still occurring in Python 3.5, IMO it sounds like a bug that needs to be raised upstream. 

Not really relevant, but I note that `generate_tokens` is no longer in the [documented interface](https://docs.python.org/3/library/tokenize.html).  Very strangely, the [v2 docs](https://docs.python.org/2/library/tokenize.html) say `tokenize.generate_tokens` is the primary interface, and note "An older entry point is retained for backward compatibility: tokenize.tokenize".  It looks like the v2 docs are wrong, and `tokenize.tokenize` is now the preferred interface.